code/modules/mob/living/carbon/human/species/station/vaurca/vaurca_subspecies.dm

```diff
@@ -73,8 +73,7 @@
 	flags = NO_SCAN | NO_SLIP
 
 	inherent_verbs = list(
-		/mob/living/carbon/human/proc/bugbite,
-		/mob/living/carbon/human/proc/hivenet
```

Comment:
Might wanna changelog this? Otherwise, looks good

Reply:
yeah this is a good, lore approved change but, changelog